packages/apps/src/configs/storages/evm-chain-storage.interface.ts

```diff
@@ -0,0 +1,22 @@
+import { MixerSize } from '@webb-dapp/react-environment';
+
+export class EvmChainStorage {
```

Comment:
I think this class should be renamed to `EvmChainMixerInfo`.  This is because most of the fields inside of this class should not be dealing with storage.